### PR TITLE
Added tests for Django 2.2-3.0 and fixes

### PR DESCRIPTION
**Pull request by dakrauth**

### CODE REVIEW

File: runtests.py [hunk 1]
```diff
@@ -12,6 +12,13 @@
 # python -Wd, or run via coverage:
 warnings.simplefilter('always', DeprecationWarning)
 
+# Introduced in Python 3.7
```
Comment (bellawoo) [inline runtests.py:15]: 🆒

**Review by bellawoo — APPROVED** [review]

File: runtests.py [hunk 2]
```diff
@@ -42,7 +49,11 @@
             'polymorphic',
             'polymorphic.tests',
         ),
-        MIDDLEWARE_CLASSES=(),
+        MIDDLEWARE=(
+            'django.contrib.auth.middleware.AuthenticationMiddleware',
+            'django.contrib.messages.middleware.MessageMiddleware',
+            'django.contrib.sessions.middleware.SessionMiddleware',
+        ),
```
Comment (defbyte) [inline runtests.py:56]: hey, I have was wondering why this is defined now when `MIDDLEWARE_CLASSES` was empty before

**Review by defbyte — COMMENTED** [review]
I've one question, the rest seems straightforward

File: runtests.py (hunk 2 above)
Comment (defbyte) [reply runtests.py:56]: David tells me this is now required!

**Review by defbyte — APPROVED** [review]

**Pull request merged**